js_modules/dagster-ui/packages/ui-core/src/runs/externalRuns.tsx

```diff
@@ -0,0 +1,16 @@
+import {Run} from '../graphql/types';
+
+export const isExternalRun = (run: Pick<Run, 'tags'>) => {
+  return run.tags.some(
+    (tag) => tag.key === 'dagster/external-job-source' && tag.value === 'airflow',
+  );
+};
+
+export const getExternalRunUrl = (run: Pick<Run, 'tags'>) => {
+  const airflowId = run.tags.find((tag) => tag.key === 'dagster/airflow_id')?.value;
+  if (!airflowId) {
+    return null;
+  }
+  // TODO: figure out external URL for airflow
```

Comment:
I think we probably need to change this:

This is meant to grab the airflow instance url and construct the view to the dag run.

Reply:
Whoops missed this one from the original. I think reasonable to pull this off of the created run as a hidden tag